Formula/dvc.rb

```diff
@@ -661,6 +661,13 @@ def install
 
   test do
     output = shell_output("#{bin}/dvc doctor 2>&1")
-    assert_match "Supports: azure, gdrive, gs, http, https, s3, ssh, oss, webdav, webdavs", output
+    assert_match "azure", output
+    assert_match "gdrive", output
+    assert_match "gs", output
+    assert_match "http", output
+    assert_match "s3", output
+    assert_match "ssh", output
+    assert_match "oss", output
+    assert_match "webdav", output
```

Comment:
For the record: the format of `dvc doctor` has changed and old test no longer worked, so I've adjusted it to be simpler.